src/libs/cleanup.sh

```diff
@@ -14,4 +14,4 @@ if [ "$TENANT_ID" == "" ]; then
 fi
 
 az login --service-principal -u $CLIENT_ID -p $CLIENT_KEY --tenant $TENANT_ID >> tmp.txt
-az cosmosdb delete --name $dbname --resource-group $resourcegroup
+az cosmosdb delete --name $dbname --resource-group $resourcegroup --debug
```

Comment:
could you please explain why `--debug` is required ?

Reply:
Due to Travis CI time limitation, a build job will be terminated when there is no log output for 10 mins.  Since the time cost of deleting a Cosmos DB instance could exceed 10 mins the job will be killed.
Adding --debug option will output some logs to prevent the job being killed. Another solution would be using 'travis_wait' command.

Reply:
I see, can we have a try on `travis_wait` ?

Reply:
It looks like that by using `travis_wait` it will hold up all our logs and output the log when the job ends. https://github.com/travis-ci/travis-ci/issues/4190

Reply:
looks [travis CI doc](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received) declares that `travis_wait` is designed for long time costing job. Can u have a try like this `travis_wait 30 set -o pipefail && mvn -P travis-ci-test clean cobertura:cobertura-integration-test | grep -v "DEBUG"` ?